server/Arcadia.Assistant.Calendar.Abstractions/CalendarEvent.cs

```diff
@@ -16,18 +17,22 @@ public class CalendarEvent
 
         public string EmployeeId { get; }
 
+        public IReadOnlyDictionary<string, string> AdditionalData { get; }
```

Comment:
i'd have it set to non-null value by default, so we avoid all stupid NRE

Reply:
Done.